packages/react-instantsearch/src/connectors/connectConfigure.js

```diff
@@ -1,7 +1,10 @@
 import createConnector from '../core/createConnector.js';
-import {omit, isEmpty, difference, keys} from 'lodash';
+import {omit, difference, keys} from 'lodash';
+import {hasMultipleIndex, getIndex, refineValue} from '../core/indexUtils';
 
-const namespace = 'configure';
+function getId() {
```

Comment:
I don't think prettier changes functions to arrow functions, it has a difference in semantics